content/authors/giuliano-maciocci/_index.md

```diff
@@ -0,0 +1,44 @@
+---
+# Display name
+title: "Giuliano Maciocci"
+
+# Username (this should match the folder name and the name on publications)
+authors:
+- "giuliano-maciocci"
+
+# Is this the primary user of the site?
+superuser: false
+
+# Role/position (e.g., Professor of Artificial Intelligence)
+role: Partnerships Lead
+
+# Organizations/Affiliations
+organizations:
+- name: 2i2c
+  url: "https://2i2c.org"
+
+# Short bio (displayed in user profile at end of posts)
+bio:
+
+# List each interest with a dash
+interests: []
+
+# Social/Academic Networking
+# For available icons, see: https://sourcethemes.com/academic/docs/page-builder/#icons
+#   For an email link, use "fas" icon pack, "envelope" icon, and a link in the
+#   form "mailto:your-email@example.com" or "#contact" for contact widget.
+social:
+- icon: envelope
+  icon_pack: fas
+  link: 'mailto:gmaciocci@2i2c.org'  # For a direct email link, use "mailto:test@example.org".
+
+
+# Enter email to display Gravatar (if Gravatar enabled in Config)
+# email: ""
+
+# Organizational groups that you belong to (for People widget)
+#   Set this to `[]` or comment out if you are not using People widget.
+user_groups:
+- Partnerships Team
+---
+
```

Comment:
@gman0909 if you want to add a short bio just reply w/ a few sentences here and we can add it, but I'll plan to merge tomorrow either way just so your name is up there